RenderStreaming~/Packages/packages-lock.json

```diff
@@ -79,15 +97,68 @@
       }
     },
     "com.unity.webrtc": {
-      "version": "2.3.3-preview",
-      "depth": 1,
+      "version": "2.4.0-exp.1",
+      "depth": 0,
       "source": "registry",
       "dependencies": {
         "com.unity.modules.jsonserialize": "1.0.0",
         "com.unity.editorcoroutines": "1.0.0"
       },
       "url": "https://packages.unity.com"
     },
+    "com.unity.xr.arcore": {
```

Comment:
Do we include the packages for these AR samples on init?

Reply:
Not needed. Removed dependencies.